pkg/plugins/plugins.go

```diff
@@ -148,13 +148,17 @@ func Get(name, imp string) (*Plugin, error) {
 	if err != nil {
 		return nil, err
 	}
-	for _, driver := range pl.Manifest.Implements {
-		logrus.Debugf("%s implements: %s", name, driver)
-		if driver == imp {
-			return pl, nil
+	if imp != "" {
```

Comment:
If we are requesting a plugin, we should always be explicit about what it implements. So if imp == "", this is an error.